website/templates/website/event.html

```diff
@@ -9,45 +9,44 @@
     <div class="event-list-title">
       <h1>{{ event.name }}</h1>
     </div>
-  </div>
-
-  <div class="event-detail-card">
+</div>
+<div class="container">
+  <div class="event-card">
     <div class="event-card-container">
       <h3 class="event-detail">Event Details</h3>
-      <p>
-        <b>Start date: </b>{{ event.start_date }}<br>
-        <b>Finish date: </b>{{ event.finish_date}}<br>
-        <b>Prerequisites: </b>{{ event.prerequisite }}<br>
-      </p>
-
+        <p>
+          <b>Start date: </b>{{ event.start_date }}<br>
+          <b>Finish date: </b>{{ event.finish_date}}<br>
+          <b>Prerequisites: </b>{{ event.prerequisite }}<br>
+        </p>
       <h3 class="event-detail">Description</h3>
-      <p>{{ event.description }}</p>
-
+        <p>{{ event.description }}</p>
       <h3 class="event-detail">Workshop Schedule</h3>
-      <table class="table">
-          <thead class="thead-dark">
-            <tr>
-              <th scope="col">Workshop Name</th>
-              <th scope="col">Workshop Time</th>
-              <th scope="col">Workshop Location</th>
-            </tr>
-          </thead>
-      {% if workshops %}
-        {% for workshop in workshops %}
-            <tbody>
-              <tr class="table-active">
-                <td>{{workshop.name }}</td>
-                <td>{{workshop.time }}</td>
-                <td>{{workshop.location }}</td>
+        <table class="table">
+            <thead class="thead-dark">
+              <tr>
+                <th scope="col">Workshop Name</th>
+                <th scope="col">Workshop Time</th>
+                <th scope="col">Workshop Location</th>
               </tr>
-            </tbody>
-        {% endfor %}
-      {% else %}
-      <p>To be confirmed</p>
-      {% endif %}
+            </thead>
+        {% if workshops %}
+          {% for workshop in workshops %}
+              <tbody>
+                <tr class="table-active">
+                  <td>{{workshop.name }}</td>
```

Comment:
Nitpick: put a space on either side of the words. I.e. `{{ workshop.name }}` instead of `{{workshop.name }}`